### PR TITLE
[FSPE-5256] Fixed documentation defects

### PR DESCRIPTION
**Pull request by janibolkvadze**
https://ctl.atlassian.net/browse/FSPE-5256

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://fbc7fffd74ff08d28f1d5cea5010c3e198660793-chi-pr.enterprise.ctl.io" target="_blank"> https://fbc7fffd74ff08d28f1d5cea5010c3e198660793-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1074/ for more details.

File: src/website/views/components/pagination/_examples.pug [hunk 1]
```diff
@@ -179,6 +179,7 @@ h3 Disabled
       </nav>
 
 h3 Truncation
+.-text When the number of pages exceeds the recommended 7 page limit, an ellipsis is used to truncate the remaining pages.
```
Comment (jllr) [inline src/website/views/components/pagination/_examples.pug:182]:
```suggestion
p.-text When the number of pages exceeds the recommended 7 page limit, an ellipsis is used to truncate the remaining pages.
```

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://2b8028a14fafd4af7199ed4e64e30a2c2d9b436d-chi-pr.enterprise.ctl.io" target="_blank"> https://2b8028a14fafd4af7199ed4e64e30a2c2d9b436d-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1075/ for more details.

**Comment by jllr** [comment]
@janibolkvadze @mattnickles Table solution works for this release, but I think that it can be improved, removing :not selector, in another ticket for next release.

File: src/chi/components/pagination/pagination.scss [hunk 2]
```diff
@@ -447,6 +447,10 @@ $icon-sizes: (
             border: 0.0625rem solid set-color(white);
             box-shadow: 0 1px 1px 0 rgba(0, 0, 0, 0.04);
           }
+
+          option {
+            color: set-color(grey, 90);
```
Comment (mattnickles) [inline src/chi/components/pagination/pagination.scss:452]: body text-color value is grey 100

```suggestion
            color: $text-color;
```

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://e7ead427d0efab9d7e8734a7fda406f545ae7b30-chi-pr.enterprise.ctl.io" target="_blank"> https://e7ead427d0efab9d7e8734a7fda406f545ae7b30-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1076/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-4sb9qz, https://ops.elasticbox.com/services/instances/i-ig0w8m, https://ops.elasticbox.com/services/instances/i-gjtqdr

**Pull request merged**